README.md

```diff
@@ -308,22 +308,50 @@ just added.
 ## API
 
 <generated_api_start />
-#### The `jest` object
+#### The Jest global environment
+
+In your test files, Jest puts each of these methods and objects into the global environment. You don't have to require or import anything to use them.
+
+  - `afterEach(fn)`
+  - `beforeEach(fn)`
+  - [`check`](https://facebook.github.io/jest/docs/api.html#check)
+  - `describe(name, fn)`
+  - [`expect(value)`](https://facebook.github.io/jest/docs/api.html#expect-value)
+  - [`gen`](https://facebook.github.io/jest/docs/api.html#gen)
+  - `it(name, fn)`
```

Comment:
Hrmph. OK for now I have not actually written any docs for `it`, which clearly we need. I'll add `test` for now but when I add docs I'll just briefly mention it's an alias for `it`.